<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeIp</class>
 <widget class="QDialog" name="ChangeIp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网关调试工具V0.0</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>189</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>IP地址：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditIP"/>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>20</width>
     <height>201</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>子网掩码：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditMask"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>177</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>网关：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditGateway"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>设置参数</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>设备别名：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>20</width>
     <height>201</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="ModificationParameter">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="GainFuntion">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>获取参数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="connect">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>断开</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ip_lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.22.66</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IP：</string>
   </property>
  </widget>
  <widget class="QLabel" name="xiaoxilan_Label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>500</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请连接服务器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Strength4G">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>4G信号强度</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="search_textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>471</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="centerOnScroll">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Memory">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>网关内存占比</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Set4G">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置4G通信</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ONlin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置以太网通信</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>常规参数</string>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>20</width>
     <height>171</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>20</width>
     <height>171</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="CllearFuntion">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GainVPNIp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>获取VPN地址</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VersionFuntion">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>版本号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rebootFuntion">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>网关重新启动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OldZigbee">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置麦希Zigbee</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NewZigbee">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置新Zigbee</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
